libraries/chain/smt_evaluator.cpp

```diff
@@ -278,6 +278,17 @@ void smt_setup_emissions_evaluator::do_apply( const smt_setup_emissions_operatio
             "New token emissions must begin no earlier than ${s} seconds after the last emission. Last emission time: ${end}",
             ("s", SMT_EMISSION_MIN_INTERVAL_SECONDS)("end", *end_time) );
       }
+      
+     for ( const auto& e : o.emissions_unit.token_unit )
+     {
+        if (smt::unit_target::is_account_name_type( e.first ))
+           {
+              std::string name = e.first;
```

Comment:
If it is a vesting account, we need to use the utility function `get_unit_target_account()` to parse the account name. See branch: `20200205-check-emission-accounts` for the unit test to verify.